lenskit/batch/_multi.py

```diff
@@ -41,14 +41,15 @@ class MultiEval:
         predict(bool):
             whether to generate rating predictions.
         recommend(int):
-            the number of recommendations to generate per user (None to disable top-N).
+            the number of recommendations to generate per user. Any false-y value (None,
+            False, 0) will disable top-n. True will generate the full recommendation list.
```

Comment:
Let's use a little syntax here - `None` and `False` are literal code values, so they should be formatted as such; the syntax for that is

    ``None``

Documentation uses reStructuredText, not Markdown, so it is a little different.  Same for `True`, and let's specifically say "The literal value `True` will generate recommendation lists of unlimited size."